manifests/init.pp

```diff
@@ -30,7 +30,9 @@
 #
 # $reverse_proxy_port::                 Reverse proxy listening port
 #
-# $rhsm_url::                           The URL that the RHSM API is rooted at
+# $rhsm_hostname::                      The hostname that the RHSM API is rooted at
+#
+# $rhsm_url::                           The URL path that the RHSM API is rooted at
```

Comment:
Looks like this parameter is badly named. 👍 for at least improving the documentation.

Reply:
Yeah, the parameter name is very misleading. `rhsm_path` or something would be better.